packages/nx-release-version/index.js

```diff
@@ -14,6 +15,8 @@ async function runSetVersion() {
 
   await updateReactNativeArtifacts(version);
 
+  spawnSync('yarn', ['install']);
```

Comment:
I think we only need to update the `yarn.lock` file?
```suggestion
  spawnSync('yarn', ['install', '--mode update-lockfile']);
```